optaplanner-quarkus-integration/optaplanner-quarkus/deployment/src/main/java/org/optaplanner/quarkus/deployment/OptaPlannerProcessor.java

```diff
@@ -615,6 +615,7 @@ private GeneratedGizmoClasses generateDomainAccessors(SolverConfig solverConfig,
         Set<String> generatedMemberAccessorsClassNameSet = new HashSet<>();
         Set<String> gizmoSolutionClonerClassNameSet = new HashSet<>();
 
+        GizmoMemberAccessorEntityEnhancer entityEnhancer = new GizmoMemberAccessorEntityEnhancer();
```

Comment:
This is my one concern around this PR.
Before this change, the enhancer's fields were static; now, I create an instance here.
Am I actually losing some cache here that was important?

Reply:
This is the processor, so this will be run only at build time. The cache should be cleared between different builds, so it is okay (and arguably good) to use an instance here.

Reply:
(additionally, this method is only called once per build in the processor)